### PR TITLE
Add minimum compatible version to --version command

### LINKED ISSUE #7477: Expose MINIMUM_COMPATIBLE_VERSION in `rasa version's` output
**Description of Problem**:
`MINIMUM_COMPATIBLE_VERSION` of the installed Rasa Open Source version is not visible to a user easily.
Having this information exposed will be helpful for users making use of the [incremental training](https://github.com/RasaHQ/rasa/issues/6971) feature.

**Overview of the Solution**:
`rasa --version` can include this information in its output.

### PR DESCRIPTION
**Pull request by joaoCeilandia**
**Proposed changes**:
- Add `MINIMUM-COMPATIBLE-VERSION` to `--version` command.
- Resolves #7477 

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @melindaloubser1 will take a look at it as soon as possible ✨

**Review by melindaloubser1 — CHANGES_REQUESTED** [review]
Thanks for contributing!
Minor thing: Could you set the spacing so that the versions all line up like before? Right now it looks like this:
```
Rasa Version     : 2.4.0
Minimum Compatible Version     : 2.2.0a1
Rasa SDK Version : 2.4.0
Rasa X Version   : None
Python Version   : 3.7.4
```
something like this:
```
Rasa Version      :         2.4.0
Minimum Compatible Version: 2.2.0a1
Rasa SDK Version  :         2.4.0
Rasa X Version    :         None
Python Version    :         3.7.4
```

**Comment by joaoCeilandia** [comment]
@melindaloubser1 done!

**Review by melindaloubser1 — APPROVED** [review]
🚀

**Comment by joaoCeilandia** [comment]
image
@melindaloubser1 this check is failling and i don't know why. Could you give me a hint?

**Comment by melindaloubser1** [comment]
It says
```
rasa/__main__.py:95:1: D103 Missing docstring in public function
```

This is because of a check introduced some months ago that requires docstrings be added to any new or changed functions.
Not sure why it picks up `main()` as changed, but you can use the existing comment on line 96 as t""he docstring e.g.
```
def main():
     """Run as standalone python application"""

File: rasa/__main__.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def print_version() -> None:
 
 
 def main() -> None:
-    # Running as standalone python application
+    """Running as standalone python application"""
```
Comment (melindaloubser1) [inline rasa/__main__.py]:
```suggestion
    """Run as standalone python application"""
```

By convention we use the imperative

**Pull request merged**